### ISSUE TITLE
Request all model checkpoins

### ISSUE THREAD

**Issue by robmarkcole** [worldstrat/worldstrat#11]
Of the 3 models:

HighResNet
SRCNN Multi-Frame
SRCNN Single-Image

It appears only HighResNet checkpoint is uploaded in https://github.com/worldstrat/worldstrat/tree/main/pretrained_model
Could I request the remaining 2?
Thanks

**Comment by jucor** [comment]
Yes, absolutely, if we still have them! @ivanorsolic, do we?

**Comment by ivanorsolic** [comment]
We sure do, I've attached them below.

Background: we've got three pre-trained models for HighResNet, SRCNN MF, SRCNN SI (triplets with different seeds).
There is no significant variance between them. The HighResNet model uploaded is just the first triplet/seed.

Likewise, the now attached SRCNN Multi-Frame and SRCNN Single-Image models are the first triplets as well.

[SRCNN Single-Image.zip](https://github.com/worldstrat/worldstrat/files/10815812/SRCNN.Single-Image.zip)
[SRCNN Multi-Frame.zip](https://github.com/worldstrat/worldstrat/files/10815803/SRCNN.Multi-Frame.zip)

**Comment by ivanorsolic** [comment]
Feel free to reopen if you need anything else related to this!

**Comment by robmarkcole** [comment]
Thank you @ivanorsolic and @jucor

**Comment by robmarkcole** [comment]
@ivanorsolic just to comment that if these were included in the model dir it would be a nice convenience for people git cloning this repo

**Comment by ivanorsolic** [comment]
I agree, that's a good point, I forgot to mention I'll was planning to
upload them to the repo this weekend.

I was in a rush yesterday, just wanted to get those checkpoints to you and
forgot.

Thank you very much for the ping, very much appreciated.
